src/Inventor/Qt/SoQtGLWidget.cpp

```diff
@@ -192,7 +192,7 @@ SoQtGLWidget::SoQtGLWidget(QWidget * const parent,
 #endif
   PRIVATE(this)->wasresized = false;
 
-  PRIVATE(this)->glformat = new QGLFormat(0);
+  PRIVATE(this)->glformat = new QGLFormat();
```

Comment:
I think we should use `QGLFormat::defaultFormat()` here as the application might have changed the default format.
See https://doc.qt.io/qt-5/qglformat.html#QGLFormat-1 and https://doc.qt.io/qt-5/qglformat.html#defaultFormat for explanation.